pkgs/build-support/ocaml/oasis.nix

```diff
@@ -28,4 +42,11 @@ buildOcaml (args // {
     prefix=$OCAMLFIND_DESTDIR ocaml setup.ml -install
     runHook postInstall
   '';
+
+  setupHook = if setupHook == null && hasSharedObjects
```

Comment:
What’s the purpose of this hook?

Reply:
Not sure to be honest. Added it because `buildOcaml` had it and I assumed the folks that wrote that had their reasons for having it. But I suppose if it turns out that it is needed (not in my uses so far) people can just send a PR and add it.